cli/internal/context/context.go

```diff
@@ -163,7 +163,7 @@ func WithGraph(rootpath string, config *config.Config) Option {
 		// We will parse all package.json's simultaneously. We use a
 		// wait group because we cannot fully populate the graph (the next step)
 		// until all parsing is complete
-		parseJSONWaitGroup := new(errgroup.Group)
+		parseJSONWaitGroup := errgroup.Group{}
```

Comment:
Should be `&errgroup.Group{}`, it would cause an issue if someone were to copy one of these

Reply:
You didn't necessarily know that this brief comment would send me on an adventure into Go internals and the history of `new` and `make`, but it did.

I had made this change intentionally for the following reasons:
- We shouldn't be passing `WaitGroup`s around. That's a recipe for disaster.
- If you decided to do it, I wanted the ergonomics of doing so to feel bad so that you *didn't* do it.
- Keep it stack allocated.

The last one was an assumption, but I didn't know for sure what it would actually do. Since you made this comment I actually went and learned all of those things and can say concretely that it ends up in the heap every time.

So the only remaining reasons to not take a pointer are ergonomics-related and, well, that's not a worthwhile reason on its own.

I'll switch it back to `parseJSONWaitGroup` being `*errgroup.Group`. 😊

Thanks for sending me down this path!